src/meta/model_v2/migration/src/m20230908_072257_init.rs

```diff
@@ -635,6 +637,13 @@ impl MigrationTrait for Migration {
                             .to(Connection::Table, Connection::ConnectionId)
                             .to_owned(),
                     )
+                    .foreign_key(
+                        &mut ForeignKey::create()
+                            .name("FK_sink_target_table_id")
+                            .from(Sink::Table, Sink::TargetTable)
+                            .to(Table::Table, Table::TableId)
```

Comment:
Unrelated to this PR, you have to manually handle it in drop cascade since it's not corded in `object_dependency` table.